### ISSUE TITLE
Request: Disable loading animations

### ISSUE THREAD

**Issue by Rockyfelle** [SimpleMobileTools/Simple-Gallery#2796]
Compared to the industry standard apps, namely Samsung Gallery and Google Gallery, Simple Gallery seemingly trades off snappyness for smoothness. It would be great to have an option to turn this off for us who like to scroll huge albums, and feel ill from the slow forced animations upon image loading.

To try and explain what I'm getting at: In Samsung/Google gallery, when you scroll down an album images will pop into existence (from a black square) as soon as they are loaded into memory, thus making the application feel snappy, of course images wont load in order due to file sizes etc, but that's not really an issue. In Simple Gallery, you essentially have a fog of war where it waits for all images to load and then slowly animates them into existence, this is particularly a problem when scrolling fast, and gives the same feeling as forced motion blurring does in game to certain people (feeling ill).

TL;DR: An option to disable loading animations for gallery scrolling.

**Comment by OmlineEditor** [comment]
if you quickly scroll down, the appearance of the image from the shadow is noticeable. This appearance of a photo from the shadow needs to be disabled
![gal](https://user-images.githubusercontent.com/13348933/227596430-ef8e78c8-390d-428c-b6d8-e513264869b0.jpg)

**Comment by rohitghali** [comment]
Wow! All this while I thought it was slow at loading. Realised it's just an animation now. Please remove it. Feels really slow.

**Comment by cyber199** [comment]
Personally I like the animation, but I can see why others wouldn't like it.  Would a toggle in settings be more preferable?

**Comment by OmlineEditor** [comment]
switching option is also a good idea. the most important thing is to turn off the animation in any way. I'm really waiting for them to make this feature

**Comment by gmasil** [comment]
Have you turned off the crossfading to see how you like it? The loading doesn't speed up whatsoever and the instant pop-in of images look a bit weird. I wouldn't use it, but it is easy to implement.
If @tibbi approves this feature I could implement it, I don't mind.
